src/step-06-testing/test/login.test.js

```diff
@@ -1,11 +1,10 @@
-import t from 'tap'
-
-const { test } = t
+import { test } from 'node:test'
+import assert from 'node:assert'
```

Comment:
Do we want to recommend/display the usage of `node:assert/strict` rather than the legacy library?

Reply:
I meant generally across all of the examples/lessons

Reply:
Oh, misread the comment. 

Make sense, we could switch to the stricter one, since I haven't used any of the `actionStrict` api methods, it should be a simple find and replace